docs/appendices/release-notes/5.10.0.rst

```diff
@@ -69,7 +69,10 @@ None
 Data Types
 ----------
 
-None
+- Added support for :ref:`NUMERIC type<type-numeric>` to the following
+  arithmetic scalar functions: :ref:`ABS<scalar-abs>`, :ref:`CEIL<scalar-ceil>`,
+  :ref:`FLOOR<scalar-floor>`, :ref:`ROUND<scalar-round>`, :ref:`EXP<scalar-exp>`
+  and :ref:`SQRT<scalar-sqrt>`.
```

Comment:
afaik one of the ROUND versions was working before with NUMERIC. 
Can we specify more precisely which version of ROUND got new signature?